test/library/validation-test.ts

```diff
@@ -233,9 +233,60 @@ describe('validations tests', () => {
 		})
 	})
 
+	describe('validationsConfig', () => {
+		it('should log an error if an unknown validation is disabled', () => {
+			validations = getValidations({ invalid: false } as any)
+			validations.validationsConfig()
+			expect(console.error).toBeCalled()
+		})
+		it('should log an error if an unknown validation is enabled', () => {
+			validations = getValidations({ invalid: false } as any)
+			validations.validationsConfig()
+			expect(console.error).toHaveBeenCalledWith(
+				expect.objectContaining({ code: 'ERR_ERL_UNKNOWN_VALIDATION' }),
+			)
```

Comment:
We should probably do this for all the tests in this file, but not as part of this PR

Reply:
👍🏾